modules/search/instant-search/components/search-filter.jsx

```diff
@@ -27,7 +27,7 @@ export default class SearchFilter extends Component {
 	constructor( props ) {
 		super( props );
 		this.filtersList = createRef();
-		this.idPrefix = `jp-instant-search-filter-${ Math.floor( Math.random() * 100 ) }`;
+		this.idPrefix = `jetpack-instant-search-filter-${ Math.floor( Math.random() * 100 ) }`;
```

Comment:
Shall we change this to use the same hook-based approach as `SearchBox` while we're in here?

Reply:
Given that SearchFilter has a lot of instance methods, I think it'd be best to migrate to a hook-based approach in a separate PR.

Do you have any pointers or guides on migrating class-based components to hook-based components?

Reply:
The switch to `uniqueId` is a good step for now I think!

> Do you have any pointers or guides on migrating class-based components to hook-based components?

Not yet - the `SearchBox` one is my first ever hook :) I also wasn't sure what the right approach was for migrating those instance methods. Let's look into it.